### ISSUE TITLE
Split terminal resize handle overlaps with text

### ISSUE THREAD

**Issue by dsboger** [gnunn1/terminix#127]
If I split terminal horizontally, It is impossible to click on the text in the extreme left of the right-side terminal. It seems the resize handle somehow overlaps with text, although visually it is ok.

**Comment by gnunn1** [comment]
I think this is actually a GTK+ bug, you can see the same behavior in gedit as follows:

a. Turn on sidebar in gedit
b. If you have it enabled, disable line numbers so the gutter disappears
c. Type text and try to select it from the left

**Comment by gnunn1** [comment]
Also, if I set the paned widehandle to true it works fine but looks ugly.
![screenshot from 2016-03-05 21-15-46](https://cloud.githubusercontent.com/assets/9358238/13551904/7b743f94-e317-11e5-8201-285833772c8b.png)

**Comment by gnunn1** [comment]
Found the GTK bug for this: https://bugzilla.gnome.org/show_bug.cgi?id=738894

Working around this may be challenging, I'll look at more over the course of the next week.

**Comment by gnunn1** [comment]
Just an FYI, I think I'm going to fix this by simply adding an option to use the wide handle. I had a look at adding padding to the VTE terminal, however it's a bit of a nightmare managing the background. basically any padding I add causes the background of the underlying panel to appear resulting in a 4px sized grey bar down the side. I looked at coloring the underlying panel and leaving the VTE transparent but there are some issues with that including when maximizing the terminal.

In short, the code was becoming way too messy for what is essentially a work-around for a GTK bug. Adding an option to use the wide handle is a much cleaner solution from a code perspective though admittedly uglier from a visual perspective. I'll also add a comment to the GTK bug about this.

I'll have this fixed for a new release this weekend.

**Comment by dsboger** [comment]
Great! I agree that the wide handle is ugly, but I also think the issue is more annoying. The (temporary?) option is a good compromise. Do not let this GTK but "infect" the code too much.

**Comment by gnunn1** [comment]
And of course you find a potential better solution right after releasing the less optimum one. Looks like just adding some CSS padding to VTE terminal works, at least on Rawhide. I'll do some testing on other distros/versions this week and see how it goes.
